docs/docker.rst

```diff
@@ -1,18 +1,87 @@
 :desc: Using Rasa NLU with Docker
+
 .. _section_docker:
 
 Running in Docker
 =================
 
-Rasa NLU docker images are provided for different backends:
+.. contents::
+
+Images
+------
+
+`Rasa NLU docker images <https://hub.docker.com/r/rasa/rasa_nlu/tags/>`_ are provided for different backends:
+
+- ``spacy``: If you use the :ref:`section_spacy_pipeline` pipeline
+- ``tensorflow``: If you use the :ref:`section_tensorflow_embedding_pipeline`
+  pipeline
+- ``mitie``: If you use the :ref:`section_mitie_pipeline` pipeline
+- ``bare``: If you want to take a base image and enhance it with your custom
+  dependencies
+- ``full`` (default): If you use components from different pre-defined pipelines
+  and want to have everything included in the image.
+
+.. note::
+
+    For the ``tensorflow`` and the ``full`` image a x86_64 CPU with AVX support
+    is required.
+
+
+Training NLU
+------------
+
+To train a NLU model you need to mount two directories into the Docker container:
+
+- a directory containing your project which in turn includes your NLU
+  configuration and your NLU training data
+- a directory which will contain the trained NLU model
 
-The host must have a x86_64 CPU with AVX support.
+.. code-block:: shell
+
+    docker run \
+        -v <project_directory>:/app/project \
+        -v <model_output_directory>:/app/model \
+        rasa/rasa_nlu:latest \
+        run \
+            python -m rasa_nlu.train \
+                -c /app/project/<nlu configuration>.yml \
+                -d /app/project/<nlu data> \
+                -o /app/model \
+                --project <nlu project name>
+
+Running NLU with Rasa Core
+--------------------------
+
+Please see this `guide <https://rasa.com/docs/core/docker_walkthrough/>`_ which
```

Comment:
Maybe we can go with 'See this guide' without 'please' - shorter and more concise :)